portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase

```diff
@@ -3731,6 +3731,689 @@ definition {
 		}
 	}
 
+	@description = "LPS-169312: Given a user belonging to Segment 1 and Segment 2
+	And the configuration to combine assets is enabled
+	When there's a variation of a dynamic collection targeting segment 1 and segment 2, and both are prioritized above the anyone variation
+	And there is an asset present in both variations**
+	Then the user will see the asset only once in Asset Publisher"
+	@priority = "3"
+	test ViewCombineContentForMultisegmentDynamicCollectionInAssetPublisher {
+		property portal.acceptance = "true";
```

Comment:
This test has priority 3 so it is not considered as acceptance. Remove this property.